Update deploy full rp service in dev documentation (#2577)
Azure/ARO-RP: docs/deploy-full-rp-service-in-dev.md

    ```

1. Create a full environment file, which overrides some default `./env` options when sourced

>     * if your ssh keypair is using `ed25519` or your key isn't named `~/.ssh/id_rsa.pub`, add `export SSH_PUBLIC_KEY=~/.ssh/id_ed25519.pub`

Reviewer: Might make more sense:
```suggestion
    * if using a public key separate from `~/.ssh/id_rsa.pub`, source it with `export SSH_PUBLIC_KEY=~/.ssh/id_separate.pub`
```
Also, not sure if we need the docs change if we add your !found function. Would like others to weigh in.
